data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,366 @@
+
+mission "Hai Legacy 1"
+	invisible
+	landing
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		not "event: hai ladybug available"
+	on offer
+		conversation
+			`As you enter the shipyard you quickly get approached by someone wearing a hood, apparently not too keen to reveal their identity. They come to address you, discreetly. Seeing you stop to listen, they take your arm and make you follow them. "Keep walking," they say, quickly taking a look around the way someone looking for a new ship would, before addressing you again. "My associates and I have a business proposition for you, if you are interested to learn more about it go to these coordinates." They give you a piece of paper containing the location, a beach almost on the other side of the planet with an instruction to not go with your ship, before quickly fading into the crowd. Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			`	You proceed to hire a motorized antigrav vehicle to go there, and arrive around sunset. You set foot after double checking you are indeed at the right coordinates, for except a cave, vegetation, sand, and, of course, the beach, you cannot see much of interest in the vicinity in the dimming light. You can't help but keep your hand close to your gun, feeling safer with its presence.`
+			`	After waiting for what seems like an eternity you start to hear whispering, and two hooded figures approach you. You start to doubt whether or not this was a good idea but it is probably too late to back down now. They come to address you, their face hidden by the now almost total obscurity. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious mood. "We are sorry for all of these complications, but the truth is we cannot afford to take risks. It has been some time since we stopped trying to understand, and started taking every precaution we could to be safe. You see, we simply do not know you, and neither do you us."`
+			choice
+				`	"Tell me more."`
+					goto tellme
+				`	"How do you know me?"`
+					goto know
+				`	(Run away).`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you. You proceed to take the antigrav back to the shipyard without any incidents.`
+				decline
+			label know
+			`	They grin, "Us knowing you is a necessity for you to land on our homeworld."`
```

Comment:
It's a bit odd to imply that this shadowy cabal of sympathizers control landing rights.

Reply:
That is not what I meant; you need reputation to land here, meaning you need to do jobs or attack the uhai
In both cases they'd know